ReactAndroid/src/main/java/com/facebook/react/bridge/ReactContext.java

```diff
@@ -196,6 +198,14 @@ public void removeActivityEventListener(ActivityEventListener listener) {
     mActivityEventListeners.remove(listener);
   }
 
+  public void addWindowFocusChangeListener(WindowFocusChangeListener listener) {
+    mWindowFocusEventListeners.add(listener);
+  }
+
+  public void removeWindowFocusChangeListener(WindowFocusChangeListener listener) {
```

Comment:
`removeWindowFocusChangeListener` is never invoked

Reply:
That's right, I left it there in case I've missed a place where I should remove all listeners